<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Monster</class>
 <widget class="QDialog" name="Add_Monster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Monster</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 55, 0, 255);</string>
  </property>
  <widget class="QLabel" name="Main_Menu_Label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Add Monster&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Add_Monster_Name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>125</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Add_Monster_Armour_Class">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Add_Monster_Challenge">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Add_Monster_Hit_Points">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>225</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>125</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Race&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>175</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Size&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Armour Class&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>225</y>
     <width>45</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hit Points&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Challenge&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Monster_Speed">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);</string>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Monster_Ability_Score">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>125</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);</string>
   </property>
   <property name="text">
    <string>Ability Scores</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Monster_Saving_Throw">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);</string>
   </property>
   <property name="text">
    <string>Saving Throws</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Add_Monster_Lore">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Lore&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Monster_Finish_Add_Monster">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>450</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);</string>
   </property>
   <property name="text">
    <string>Add Monster</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Saving_Throws_Foreign_Key">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>50</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Abilities_Foreign_Key">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>30</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Speed_Foreign_Key">
   <property name="geometry">
    <rect>
     <x>575</x>
     <y>10</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Monster_Populate">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>175</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);</string>
   </property>
   <property name="text">
    <string>Populate Foreign Keys</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Add_Monster_Race_Dropdown">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Add_Monster_Size_Dropdown">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>174</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(140, 140, 140);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Add_Monster_Speed</sender>
   <signal>clicked()</signal>
   <receiver>Add_Monster</receiver>
   <slot>Monster_Speed_Link()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>963</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Add_Monster_Ability_Score</sender>
   <signal>clicked()</signal>
   <receiver>Add_Monster</receiver>
   <slot>Monster_Ability_Scores_Link()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>805</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Add_Monster_Saving_Throw</sender>
   <signal>clicked()</signal>
   <receiver>Add_Monster</receiver>
   <slot>Monster_Saving_Throws_Link()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>776</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Add_Monster_Populate</sender>
   <signal>clicked()</signal>
   <receiver>Add_Monster</receiver>
   <slot>Import_Foreign_Key()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>184</y>
    </hint>
    <hint type="destinationlabel">
     <x>904</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Add_Monster_Finish_Add_Monster</sender>
   <signal>clicked()</signal>
   <receiver>Add_Monster</receiver>
   <slot>Add_Monster_Add_To_DB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>555</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>1121</x>
     <y>518</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Monster_Speed_Link()</slot>
  <slot>Monster_Ability_Scores_Link()</slot>
  <slot>Monster_Saving_Throws_Link()</slot>
  <slot>Import_Foreign_Key()</slot>
  <slot>Add_Monster_Add_To_DB()</slot>
 </slots>
</ui>
